components/targets/source-code-artifact/component.yaml

```diff
@@ -0,0 +1,34 @@
+name: "source-code-artifact"
+description: "Downloads archived files in zip, tag or targz format from a plain http or s3 compatible API"
+type: "target"
+parameters:
```

Comment:
we should support one more parameter to allow users to set the type of archive explicitly for some artefact that doesn't have a suffix